feat(kinput): slot tooltip [khcp-7093] (#1351)
Kong/kongponents: docs/components/label.md

<KInput id="service"/>
```

## Slots

> - `tooltip` - Rather than using the `help` or `info` props, if you want to style the tooltip content you can use this slot.

Reviewer: ```suggestion
- `tooltip` - Rather than using the `help` or `info` props, if you need to utilize HTML in the tooltip, you may use the `tooltip` slot.
```

:::tip Note:

> By default, when using the `tooltip` slot the `info` `KIcon` will be shown, but you can get the `help` icon to display instead by setting the `help` prop to any non-empty string value.

Reviewer: ```suggestion
When utilizing the `label-tooltip` slot, the `info` `KIcon` will be shown by default. To utilize the the `help` icon instead, set the `label-attributes` `help` property to any non-empty string value.
```

:::

<KLabel help="true">
  My Tooltip
  <template #tooltip>Brings all the <code>devs</code> to the yard</template>
</KLabel>
<KInput />

```html

> <KLabel help="true">

Reviewer: nit: can we remove the `help` attribute in the code and example since we're using the slot?

Author: It's there because this example is showing how you can get the `help` icon (instead of `info`) when using the slot.

Reviewer: Ah that makes sense. Maybe add a note about how to specify the icon in both of the docs pages?
